test/gracenote/response_test.rb

```diff
@@ -97,15 +86,23 @@ module Gracenote
         <GN_ID>153857542-F5B1994A6C9EBF79D7FFB02EDF189A1D</GN_ID>
         <TITLE>チョコレイト・ディスコ</TITLE>
       </TRACK>
+      <URL TYPE="COVERART" SIZE="MEDIUM" WIDTH="450" HEIGHT="390">http://example.com/cover.jpg</URL>
     </ALBUM>
   </RESPONSE>
 </RESPONSES>
         XML
         )
 
-        response = Gracenote::Client.new(client_id: "test", user_id: "test").search(artist: "Perfume", album: "GAME", track: "チョコレイト・ディスコ")
+        client = Gracenote::Client.new(client_id: "test", user_id: "test")
+        response = client.search(artist: "Perfume", album: "GAME", track: "チョコレイト・ディスコ")
         assert { response.ok? == true }
-        assert { response.params["album"]["gn_id"] == "153857537-C0E89B7AA71C102BD8986ADD02B0DE9C" }
+        assert { response.album.gn_id == "153857537-C0E89B7AA71C102BD8986ADD02B0DE9C" }
+        assert { response.album.track.track_num == 5 }
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)